.github/workflows/linux.yml

```diff
@@ -96,7 +96,7 @@ jobs:
           - name: GCC, Debian 10, ARMv7
             os: ubuntu-20.04
             build_flags: -Dbuildtype=debug -Dexperimental_voodoo_support=true --cross-file dosbox-cross
-            max_warnings: 2
+            max_warnings: 21
```

Comment:
That's a bit unfortunate to have to increase the max warnings. Going from 2 to 21 is quite significant. Ideally, we want to go from 2 to 0 😄 Can't you find a better workaround so we can keep it at 2?

Reply:
I did read the commit description, but I'm wondering if there's a way to keep `max_warnings` at 2 while keeping your change 😄 

E.g. if you can use some pragma or "magic comment" to make these problematic compilers happy to ignore these 20 warnings, that's a much better workaround than globally increasing the number of allowed warnings for all compilers from 2 to 21 wholesale. That's a bit too heavy-handed approach, and I'd actually prefer not to do this change at all and keep the `max_warnings` level at 2.

Reply:
Ok kinda missed that, my bad. I thought it's for all Linux builds. If others are happy with this, I think it's acceptable compromise perhaps. @FeralChild64 @shermp ?